apps/opencs/view/world/scriptsubview.cpp

```diff
@@ -198,26 +198,38 @@ void CSVWorld::ScriptSubView::useHint (const std::string& hint)
     if (hint.empty())
         return;
 
-    if (hint[0]=='l')
-    {
-        std::istringstream stream (hint.c_str()+1);
-
-        char ignore;
-        int line;
-        int column;
-
-        if (stream >> ignore >> line >> column)
+    size_t line = 0, column = 0;
+    std::istringstream stream (hint.c_str()+2);
```

Comment:
Okay, i just thought that hint is internal thing, and we could rely on his consistency with format [char]: some info.
But I can't understand how this check works: "if (stream >> ignore >> line >> column)". >> operator returns istream&, that somehow casts to bool, but according to [this](http://www.cplusplus.com/reference/sstream/istringstream/?kw=istringstream), bool operator was introduced in c++11.